uwmac0030.xml

```diff
@@ -170,7 +176,7 @@
       <head>Related Material in the UWM Libraries</head>
```

Comment:
Can you confirm that Simone Ferro is still the chair? If not, this list should be updated